tempus_fugit_minecraft/window.py

```diff
@@ -201,35 +201,35 @@ def on_key_press(self, symbol: int, modifiers: int) -> None:
             return
 
         if symbol in self.num_keys:
-            index = (symbol - self.num_keys[0]) % len(self.model.player.inventory)
-            self.model.handle_change_active_block(index)
+            index = (symbol - self.num_keys[0]) % len(self.game_model.player.inventory)
+            self.game_model.handle_change_active_block(index)
             return
 
         if symbol in [key.Q, key.E]:
             increase_speed = symbol == key.Q
             increase_jump_speed = symbol == key.Q
```

Comment:
Guess we need to change these too, probably to increase_player_speed or increase_player_jump_speed to be more specific. Though it also means we will need to change the name of the functions too.

Reply:
I agree to make it more clear Increase_speed should be more specific like increase_walking_speed or increase_flying_speed

Reply:
Renamed to `..._walk_speed` to clarify that we're interacting with the walking speed.